### PR TITLE
Add Elegoo OrangeStorm Giga

### LINKED ISSUE #6256: OrangeStorm Giga profile
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Hello, could we add a new printer profile for OrangeStorm Giga? Link here: https://www.elegoo.com/products/orangestorm-giga

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

CleanShot 2024-07-26 at 16 13 26@2x

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cochcoder**
Closes #6256

Thanks to @tlhintoq for providing the profiles in a easily downloadable manner.

### CODE REVIEW

**Comment by cochcoder** [comment]
@SoftFever 

Do you have any idea what could be causing this error? I don't see any syntax errors.

File: resources/profiles/Elegoo.json [hunk 1]
```diff
@@ -817,6 +953,22 @@
...
+			"name": "Elegoo OrangeStorm Giga 1.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 1.0 nozzle.json"
+		},
+		{
+			"name": "Elegoo OrangeStorm Giga 6.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 6.0 nozzle.json"
+		},
+		{
+			"name": "Elegoo OrangeStorm Giga 8.0 nozzle",
+			"sub_path": "machine/Elegoo OrangeStorm Giga 8.0 nozzle.json"
```
Comment (arough007) [inline resources/profiles/Elegoo.json:971]: This seems to cause the error, as it's looking for the wrong files.
Classic typo :D 

```suggestion
		},
		{
			"name": "Elegoo OrangeStorm Giga 0.4 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 0.4 nozzle.json"
		},
		{
			"name": "Elegoo OrangeStorm Giga 1.0 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 1.0 nozzle.json"
		},
		{
			"name": "Elegoo OrangeStorm Giga 0.6 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 0.6 nozzle.json"
		},
		{
			"name": "Elegoo OrangeStorm Giga 0.8 nozzle",
			"sub_path": "machine/Elegoo OrangeStorm Giga 0.8 nozzle.json"
```

Comment (cochcoder) [reply resources/profiles/Elegoo.json:971]: Thank you so much! You have no idea how much time I wasted looking at that file and not finding anything!

Comment (arough007) [reply resources/profiles/Elegoo.json]: Guessed as much by your increasingly desperate commit messages :D

Comment (cochcoder) [reply resources/profiles/Elegoo.json]: lol

Comment (SoftFever) [reply resources/profiles/Elegoo.json]: Hi @cochcoder, I apologize for not being available. I was traveling and then fell ill for a while. I'm really grateful that @arough007 could step in to help. Thank you both so much!

**Review by SoftFever — APPROVED** [review]

File: resources/profiles/Elegoo.json (hunk 1 above)
Comment (cochcoder) [reply resources/profiles/Elegoo.json]: > Hi @cochcoder, I apologize for not being available. I was traveling and then fell ill for a while. I'm really grateful that @arough007 could step in to help. Thank you both so much!

It's not a issue at all! Thank you for all your work on this slicer and thank you to @arough007 again for finding the source of this error!

**Comment by tlhintoq** [comment]
@cochcoder 

Ummm... After installing 2.2.0 release the resources|elegoo folder is a mess.
It has the nicely broken down folders by machine type (my previous install and I thought you were going to implement them, and then all the machine types again loose.
Was that deliberate?

**Comment by GitZapf** [comment]
In addition to the organizational changes, the profiles that now exist in orcaslicer 2.2 don't match settings-wise at all to the ones that @tlhintoq provided.

With a fresh orcaslicer 2.2 and the orangestorm giga profile contained therein, a 1000% voron cube at a .2mm profile (with perimeters, tops/bottoms, brim and infill to match the elegoo cura profile) the print takes 8+ days compared to elegoo cura (4.8, the one on the usb drive)'s 3ish. If I overwrite the elegoo folder with the only that softfever is referencing in that commit (https://github.com/SoftFever/OrcaSlicer/issues/6256#issuecomment-2362430471 ), the print estimate drops down to more closely resemble the elegoo cura estimate.

Screenshot 2025-01-12 110522
Screenshot 2025-01-12 110620
Screenshot 2025-01-12 113055

**Pull request merged**